squirrel.frontier/src/main/java/org/dice_research/squirrel/queue/scorecalculator/IUriScoreCalculator.java

```diff
@@ -0,0 +1,16 @@
+package org.dice_research.squirrel.queue.scorecalculator;
+
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+
+/**
+ *  Interface for getting duplicity score of a Uri.
+ */
+public interface IUriScoreCalculator {
+
+    /**
+     * Returns the score based on specific score calculation implementation
```

Comment:
This comment is already much better since it concentrates on the signature of the method. However, it would be a little bit better if you would include that the score is calculated for the given URI, e.g, "Returns a score for the given URI based on the implemented scoring function."